libs/ui/checkbox/helm/src/lib/hlm-checkbox.component.ts

```diff
@@ -0,0 +1,120 @@
+import { booleanAttribute, Component, computed, EventEmitter, forwardRef, Input, Output, signal } from '@angular/core';
+import { NG_VALUE_ACCESSOR } from '@angular/forms';
+import { BrnCheckboxComponent, indeterminateBooleanAttribute } from '@spartan-ng/ui-checkbox-brain';
+import { hlm } from '@spartan-ng/ui-core';
+import { ClassValue } from 'clsx';
+import { HlmCheckboxCheckIconComponent } from './hlm-checkbox-checkicon.component';
+
+export const HLM_CHECKBOX_VALUE_ACCESSOR = {
+	provide: NG_VALUE_ACCESSOR,
+	useExisting: forwardRef(() => HlmCheckboxComponent),
+	multi: true,
+};
+
+@Component({
+	selector: 'hlm-checkbox',
+	standalone: true,
+	imports: [BrnCheckboxComponent, HlmCheckboxCheckIconComponent],
+	template: `
+		<brn-checkbox
+			[class]="_computedClass()"
+			[checked]="_checked()"
+			(changed)="_handleChange()"
+			(touched)="_onTouched()"
+			[disabled]="_disabled()"
+			[id]="id"
+		>
+			<hlm-checkbox-checkicon [iconName]="_checkIconName()" />
+		</brn-checkbox>
+	`,
+	host: {
+		class: 'contents',
+		'[attr.id]': 'null',
+		'[attr.aria-label]': 'null',
+		'[attr.aria-labelledby]': 'null',
+		'[attr.aria-describedby]': 'null',
+	},
+	providers: [HLM_CHECKBOX_VALUE_ACCESSOR],
```

Comment:
We should still provide the default icon here. Can you add `provideIcons({radixCheck})`

Reply:
It is already provided in the checkicon component. 

libs/ui/checkbox/helm/src/lib/hlm-checkbox-checkicon.component.ts

Reply:
You're absolutely right! I am an idiot 😭 😄